### DISCUSSION TITLE
How to build ANE for AIR 50

### DISCUSSION THREAD

**Discussion by abibik1** [airsdk/Adobe-Runtime-Support#2390] [General]
I have an old Android ANE, built by me probably 7 or 10 years ago. When I tried to use that ANE with the app converted to AIR 50 the ANE stopped working. Naturally, I tried to re-compile it, updating Android SDK as well to the latest... But things turned out to be not so easy...
When I built the ANE for the first time, I used the User Guide from Adobe, Flash Builder and Eclipse. None of these instructions are applicable anymore. Is there any modern guide on building ANE? Right now I an stuck on building JAR using Android Studio which includes third-party JAR's and resources.

Anyone?

**Comment by balaedit** [comment]
Please share so that I could help. balaedit1987@gmail.com

**Comment by mrfrasier** [comment]
If the JAR is throwing a wrench in your progress, the AAR that Android Studio generates is an archive. You can open it up and grab the classes.jar file out of it.

**Comment by Joseph-Cat** [comment]
Bonjour, 
Moi aussi j'ai quelque chose à faire et pour cela je dois créer un ANE mais je ne trouve pas de formation pour pouvoir en créer un.
Merci déjà pour vos réponses.
